dev/io.openliberty.transport.ssl_fat/publish/files/SSLOptions.xml

```diff
@@ -0,0 +1,53 @@
+<!--
+    Copyright (c) 2016 IBM Corporation and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License 2.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-2.0/
+    
+    SPDX-License-Identifier: EPL-2.0
+   
+    Contributors:
+        IBM Corporation - initial API and implementation
+ -->
+<server description="FAT for channel SSL - alternate SSLConfig">
+
+    <include optional="true" location="../fatTestPorts.xml"/>
```

Comment:
Should we just use fatTestCommon.xml rather than the fatTestPorts.xml since we're defining the endpoint in these server.xml and then add the portOpenRetries so we don't lose that from fatTestPorts.xml?

I believe if you look at the logs we're going to see messages about duplicate endpoints or something like that similar to what I saw / fixed in : https://github.com/OpenLiberty/open-liberty/issues/31081